parallelization_test/parallelization-benchmark.ipynb

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
<p>could you add a brief comment in the Markdown cell where this data (benchmark_download_times.csv') is coming from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>